ExpressAPI/db.js

```diff
@@ -10,4 +10,16 @@ const db = mysql.createConnection({
 db.connect(function(err){
     if(err) throw err;
     console.log("Connected to MySql");
-})
\ No newline at end of file
+})
+
+exports.getCourses = function(callback) {
```

Comment:
yeah aware of this double effort between Eoin and myself so will just need to remember that when merging the later API stuff